andela/ah-frontend-realers
# User is able to view and edit profiles

#### What does this PR do?

Enables users to signup

#### Description of task completed
* created actions, reducers and tests relevant for viewing a user profile or editing it.

#### How should this be manually tested on your local machine
* Do a `git fetch origin ft-view-and-edit-profile-166187819`
* Do a `git checkout ft-view-and-edit-profile-166187819`
* Run this command `$ npm install`
* Run this command `$ npm start`
*make sure you sign up and login
* update the url in the browser by adding `/profiles` for viewing profiles and `/editprofile` for editing profile.
*You should be able to view your profile and edit it too.




[finishes #166187819](https://www.pivotaltracker.com/n/projects/2350340/stories/1166187819)
